Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/DatabaseException.java

```diff
@@ -0,0 +1,24 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+import com.google.gson.JsonObject;
+
+public class DatabaseException extends ServerException {
+
+    String type;
+    String message;
+
+    public DatabaseException(String error, JsonObject details){
+        super(error);
+        type= details.get("type").getAsString();
+        message=details.get("message").getAsString();
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    @Override
+    public String getMessage() {
+        return message;
+    }
```

Comment:
See comment at ServerException. Override shouldn't be needed when the comment there is fixed.